### PR TITLE
fixed slides proportions when loading on different size canvas

### PR DESCRIPTION
**Pull request by Aror**

### CODE REVIEW

File: src/components/Editor/index.jsx [hunk 1]
```diff
@@ -38,7 +38,11 @@ const Editor = ({ getNewSlide, deleteSlide, activePresentation }) => {
   }
 
   const addText = () => {
-    canvas.add(new fabric.IText('Edit Me'))
+    let text = new fabric.IText('Edit Me')
+    text.on('dblclick', function(e) {
```
Comment (ericbastarache) [inline src/components/Editor/index.jsx]: what is the purpose of the double click handler?

File: src/components/Presentation/index.jsx [hunk 2]
```diff
@@ -51,9 +50,26 @@ const Presentation = ({
     return canvas
   }
 
+  const updateSlideWithNewResolution = (canvasObjects, slideWidth) => {
+    if (canvas.width != slideWidth) {
+      let scaleMultiplier = canvas.width / slideWidth
+      var objects = canvasObjects
```
Comment (ericbastarache) [inline src/components/Presentation/index.jsx]: can we use es6 for these variables? we should avoid mixing es5 and es6

Comment (Aror) [reply src/components/Presentation/index.jsx]: Oops, lefts over from copy and paste

File: src/components/Editor/index.jsx (hunk 1 above)
Comment (Aror) [reply src/components/Editor/index.jsx]: testing, its removed in the next push

**Pull request closed without merging**